webtau-http-groovy/src/test/groovy/com/twosigma/webtau/http/HttpGroovyTest.groovy

```diff
@@ -314,6 +334,11 @@ class HttpGroovyTest extends HttpTestBase {
             // assertions go here
         }
 
+        http.patch("/end-point", http.header('Accept', 'application/octet-stream'),
```

Comment:
Just curios about `patch` specs, what is the intended behavior of patching with an octet stream?

Reply:
I guess it depends on what resource you are patching and how the server interprets/applies the patch.  The RFC (https://tools.ietf.org/html/rfc5789#section-2.2) is vague in that regard; it has this example:
```   
PATCH /file.txt HTTP/1.1
   Host: www.example.com
   Content-Type: application/example
   If-Match: "e0023aa4e"
   Content-Length: 100

   [description of changes]```